alacritty/src/event.rs

```diff
@@ -542,6 +544,13 @@ impl<N: Notify + OnResize> Processor<N> {
                             processor.ctx.display_update_pending.font = Some(font);
                         }
 
+                        #[cfg(not(any(target_os = "macos", windows)))]
+                        {
+                            if processor.ctx.event_loop.is_wayland() {
+                                processor.ctx.window.set_wayland_theme(&config.colors);
+                            }
+                        }
```

Comment:
This is getting kinda big, maybe we should consider moving the `ConfigReload` handling to a separate method?